tests/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -737,4 +765,92 @@ public function testBanner(): void {
     );
   }
 
+  /**
+   * Test contact paragraph rendering.
+   *
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  public function testContact(): void {
```

Comment:
I would make a new test class for this. We are doubling the requirements due this one test which will run for all the other tests in this class.